src/index.html

```diff
@@ -7,8 +7,52 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet"
+          href="./normalize.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#Home">
+        <img class="logo image"
+             src="images/logo.png" alt="logo">
```

Comment:
якщо три і більше атрибутів в тега то потрібно всі ці атрибути переносити на новий рядок
```suggestion
        <img
          class="logo image"
          src="images/logo.png"
          alt="logo"
        >
```